wagtailio/templates/base.html

```diff
@@ -168,15 +154,7 @@
 
         <script src="https://platform.twitter.com/widgets.js" async></script>
         <script async defer src="https://buttons.github.io/buttons.js"></script>
-
-        {% compress js %}
-            <script src="{% static 'js/vendor/jquery.min.js' %}"></script>
-            <script src="{% static 'js/vendor/js-cookie.min.js' %}"></script>
-            <script src="{% static 'js/vendor/owl.carousel.js' %}"></script>
-            <script src="{% static 'js/vendor/jQuery.headroom.min.js' %}"></script>
-            <script src="{% static 'js/vendor/headroom.min.js' %}"></script>
-            <script src="{% static 'js/main.js' %}"></script>
-        {% endcompress %}
+        <script src="{% static 'js/main.js' %}"></script>
```

Comment:
At Four Digits we use https://django-manifest-loader.readthedocs.io/en/latest/ for invalidating the browser cache. Might be worth a look.

The Webpack bundle names can be hashed. Django Manifest Loader reads the manifest file and provides the right URLs.

Might be worth a look.

Here is a public example:
- https://github.com/wagtail/guide/blob/main/apps/guide/settings/base.py#L31
- https://github.com/wagtail/guide/blob/main/apps/guide/settings/base.py#L178-L180
- https://github.com/wagtail/guide/blob/main/apps/guide/settings/production.py#L5
- https://github.com/wagtail/guide/blob/main/apps/frontend/templates/base.html#L1
- https://github.com/wagtail/guide/blob/main/apps/frontend/templates/base.html#L20
- https://github.com/wagtail/guide/blob/main/apps/frontend/templates/base.html#L37

Reply:
Looks good! I've added this in. Thanks for providing the links : )